bricks-model/src/main/java/com/github/ingogriebsch/bricks/model/lombok.config

```diff
@@ -0,0 +1,2 @@
+lombok.fieldDefaults.defaultPrivate = true
+lombok.fieldDefaults.defaultFinal = false
```

Comment:
Is it possible to configure Lombok to act in a specific way for a specific package? Or does the config now count for the whole module?

Reply:
no, it is just the package. lombok has this habit of bubbeling up until it finds a config that asks it to "stopBubbeling"

Reply:
https://projectlombok.org/features/configuration

Reply:
that is why i like to keep it near to the code, otherwise it'll be hard to know which defaults apply where.

Reply:
Always thought, that only one lombok.config per project is possible. Nice to know that this is working. Putting the config file into the matching resource folder is for me 'near to the code'. :)